.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
     // Check if imports actually resolve
     settings: {
         'import/resolver': {
+            node: {},
```

Comment:
This is required so node dependencies will resolve. Passing an empty object sets default options from what I understood from github. Having trouble finding the original issue I read that from but here are the docs, https://github.com/benmosher/eslint-plugin-import#settings